src/codewhisperer/activation.ts

```diff
@@ -4,57 +4,58 @@
  */
 
 import * as vscode from 'vscode'
-import { getTabSizeSetting } from '../shared/utilities/editorUtilities'
-import { KeyStrokeHandler } from './service/keyStrokeHandler'
-import * as EditorContext from './util/editorContext'
-import * as CodeWhispererConstants from './models/constants'
-import { getCompletionItems } from './service/completionProvider'
-import { vsCodeState, ConfigurationEntry } from './models/model'
-import { invokeRecommendation } from './commands/invokeRecommendation'
-import { acceptSuggestion } from './commands/onInlineAcceptance'
-import { resetIntelliSenseState } from './util/globalStateUtil'
-import { CodeWhispererSettings } from './util/codewhispererSettings'
-import { ExtContext } from '../shared/extensions'
 import { TextEditorSelectionChangeKind } from 'vscode'
-import { CodeWhispererTracker } from './tracker/codewhispererTracker'
-import * as codewhispererClient from './client/codewhisperer'
-import { runtimeLanguageContext } from './util/runtimeLanguageContext'
-import { getLogger } from '../shared/logger'
 import { isCloud9 } from '../shared/extensionUtilities'
+import { ExtContext } from '../shared/extensions'
```

Comment:
Can we undo these formatting changes, its causing too much noise

Reply:
sure. Updating it